client/index.js

```diff
@@ -1,4 +1,5 @@
 // View-specific code
 var pathname = window.location.pathname
 if (pathname === '/') require('./views/home')()
-if (pathname === '/create') require('./views/create')()
+if (/\/create\/?/.test(pathname)) require('./views/create')()
+if (/\/desktop\/?/.test(pathname)) require('./views/desktop')()
```

Comment:
It didn't really. The `/create` page was a work-in-progress from the very first version of the site (years ago!). It should probably be removed!